.travis.yml

```diff
@@ -65,11 +65,16 @@ stage_osx:
     - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$DEPS_INSTALL_PREFIX/lib
     - export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:$DEPS_INSTALL_PREFIX
     - export CMAKE_BUILD_OPTIONS=""
+    # Define brewInstallOrUpgrade function
+    - sh .ci/brewInstallOrUpgrade.sh
```

Comment:
Usually in these cases you can use source to import a function to the running interpreter

Reply:
You are right! I fixed that in [HDE](https://github.com/robotology/human-dynamics-estimation/pull/216/commits/22a43dafece2d8d1085757a7cc83758d4a2c4170) yesterday and I did the same error here. Thanks